keen/scoped_keys.py

```diff
@@ -2,8 +2,15 @@
 import json
 import os
 import six
-from Crypto.Cipher import AES
 
+try:
+    from Crypto.Cipher import AES
+except ImportError as ie:
+    ie.args = (ie.args[0] + ' -- Scoped Keys are deprecated in favor'
+               ' of Access Keys. To use Scoped Keys anyway, run: '
+               '`pip install pycryptodome` and try this again.',)
+    raise ie
```

Comment:
This will blow away the original stack, right? `raise ie` in python 2 will blow away the stack anyway since you have to use the 3-arg `raise`, and in python 3, by replacing the entire `args` tuple we're disregarding any information previously stored on the exception except the message, right? In this case, maybe the message is enough, but if not, we could use the `six` module's `reraise` which I think forwards to the `future` utils in python 2 and uses the `raise ... from ...` facility in python 3. Just a thought.